src/scss/dp/overrides/components/_icons.scss

```diff
@@ -0,0 +1,32 @@
+@mixin small-icon {
+    height: 16px;
+    width: 20px;
+}
+
+$sprite: 'https://cdn.ons.gov.uk/assets/images/icon-sprite/v2/icon-sprite.png';
```

Comment:
Could remove `$sprite` as a duplicate and have `$spritePng` and `$spriteSvg` instead